### DISCUSSION TITLE
vue-cesium and Quasar?

### DISCUSSION THREAD

**Discussion by tob-at-lovelace** [zouyaoji/vue-cesium#497] [Q&A] [answered]
Hi all. Has anyone tried this? I noticed one question a while back that mentioned some sort of CSS conflict. Has anyone tried creating a default Quasar app (e.g. from [here](https://quasar.dev/start/quick-start) ) and adding vue-cesium to it recently? I'm about to dive in myself but thought I'd see if anyone has something already working.

Thanks!

**Comment by zouyaoji** [comment, marked as answer]
https://github.com/zouyaoji/vue-cesium-demo

**Reply by tob-at-lovelace** [reply]
Oh wow, I have no idea how I missed the existence of this demo project. This is absolutely amazing. Thank you!!!
